Requirement already satisfied: transformers in c:\users\as920\appdata\roaming\python\python312\site-packages (4.44.0)
Requirement already satisfied: pandas in c:\users\as920\appdata\roaming\python\python312\site-packages (2.0.3)
Requirement already satisfied: nltk in c:\users\as920\appdata\roaming\python\python312\site-packages (3.8.1)
Collecting textblob
  Obtaining dependency information for textblob from https://files.pythonhosted.org/packages/02/07/5fd2945356dd839974d3a25de8a142dc37293c21315729a41e775b5f3569/textblob-0.18.0.post0-py3-none-any.whl.metadata
  Downloading textblob-0.18.0.post0-py3-none-any.whl.metadata (4.5 kB)
Requirement already satisfied: flask in c:\users\as920\appdata\roaming\python\python312\site-packages (2.3.2)
Collecting pyngrok
  Obtaining dependency information for pyngrok from https://files.pythonhosted.org/packages/9e/09/8338c63fe2d92313b7819f38c235746689d88052b112d8717bf45dcb3570/pyngrok-7.2.0-py3-none-any.whl.metadata
  Downloading pyngrok-7.2.0-py3-none-any.whl.metadata (7.4 kB)
Requirement already satisfied: filelock in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (3.13.4)
Requirement already satisfied: huggingface-hub<1.0,>=0.23.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (0.24.5)
Requirement already satisfied: numpy>=1.17 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (1.26.4)
Requirement already satisfied: packaging>=20.0 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (23.1)
Requirement already satisfied: pyyaml>=5.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (6.0.1)
Requirement already satisfied: regex!=2019.12.17 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (2023.6.3)
Requirement already satisfied: requests in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (2.31.0)
Requirement already satisfied: safetensors>=0.4.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (0.4.4)
Requirement already satisfied: tokenizers<0.20,>=0.19 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (0.19.1)
Requirement already satisfied: tqdm>=4.27 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (4.65.0)
Requirement already satisfied: python-dateutil>=2.8.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from pandas) (2023.3)
Requirement already satisfied: tzdata>=2022.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from pandas) (2023.3)
Requirement already satisfied: click in c:\users\as920\appdata\roaming\python\python312\site-packages (from nltk) (8.1.6)
Requirement already satisfied: joblib in c:\users\as920\appdata\roaming\python\python312\site-packages (from nltk) (1.3.1)
Requirement already satisfied: Werkzeug>=2.3.3 in c:\users\as920\appdata\roaming\python\python312\site-packages (from flask) (2.3.6)
Requirement already satisfied: Jinja2>=3.1.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from flask) (3.1.2)
Requirement already satisfied: itsdangerous>=2.1.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from flask) (2.1.2)
Requirement already satisfied: blinker>=1.6.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from flask) (1.6.2)
Requirement already satisfied: colorama in c:\users\as920\appdata\roaming\python\python312\site-packages (from click->nltk) (0.4.6)
Requirement already satisfied: fsspec>=2023.5.0 in c:\users\as920\appdata\roaming\python\python312\site-packages (from huggingface-hub<1.0,>=0.23.2->transformers) (2024.3.1)
Requirement already satisfied: typing-extensions>=3.7.4.3 in c:\users\as920\appdata\roaming\python\python312\site-packages (from huggingface-hub<1.0,>=0.23.2->transformers) (4.11.0)
Requirement already satisfied: MarkupSafe>=2.0 in c:\users\as920\appdata\roaming\python\python312\site-packages (from Jinja2>=3.1.2->flask) (2.1.3)
Requirement already satisfied: six>=1.5 in c:\users\as920\appdata\roaming\python\python312\site-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from requests->transformers) (3.2.0)
Requirement already satisfied: idna<4,>=2.5 in c:\users\as920\appdata\roaming\python\python312\site-packages (from requests->transformers) (3.4)
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from requests->transformers) (2.0.4)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\as920\appdata\roaming\python\python312\site-packages (from requests->transformers) (2023.7.22)
Downloading textblob-0.18.0.post0-py3-none-any.whl (626 kB)
   ---------------------------------------- 626.3/626.3 kB 1.0 MB/s eta 0:00:00
Downloading pyngrok-7.2.0-py3-none-any.whl (22 kB)
Installing collected packages: pyngrok, textblob
  WARNING: The scripts ngrok.exe and pyngrok.exe are installed in 'C:\Users\as920\AppData\Local\Programs\Python\Python312\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed pyngrok-7.2.0 textblob-0.18.0.post0
WARNING: There was an error checking the latest version of pip.
Traceback (most recent call last):
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\transformers\utils\import_utils.py", line 1603, in _get_module
    return importlib.import_module("." + module_name, self.__name__)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\importlib\__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1381, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1354, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1325, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 929, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 994, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\transformers\pipelines\__init__.py", line 26, in <module>
    from ..image_processing_utils import BaseImageProcessor
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\transformers\image_processing_utils.py", line 20, in <module>
    from .image_processing_base import BatchFeature, ImageProcessingMixin
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\transformers\image_processing_base.py", line 45, in <module>
    from PIL import Image
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\PIL\Image.py", line 82, in <module>
    from . import _imaging as core
ImportError: cannot import name '_imaging' from 'PIL' (C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\PIL\__init__.py)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\server.py", line 1, in <module>
    from transformers import pipeline
  File "<frozen importlib._bootstrap>", line 1406, in _handle_fromlist
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\transformers\utils\import_utils.py", line 1593, in __getattr__
    module = self._get_module(self._class_to_module[name])
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\transformers\utils\import_utils.py", line 1605, in _get_module
    raise RuntimeError(
RuntimeError: Failed to import transformers.pipelines because of the following error (look up to see its traceback):
cannot import name '_imaging' from 'PIL' (C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\PIL\__init__.py)
Script finished. Check script_log.txt for any errors.
Requirement already satisfied: transformers in c:\users\as920\appdata\roaming\python\python312\site-packages (4.44.0)
Requirement already satisfied: pandas in c:\users\as920\appdata\roaming\python\python312\site-packages (2.0.3)
Requirement already satisfied: nltk in c:\users\as920\appdata\roaming\python\python312\site-packages (3.8.1)
Requirement already satisfied: textblob in c:\users\as920\appdata\local\programs\python\python312\lib\site-packages (0.18.0.post0)
Requirement already satisfied: flask in c:\users\as920\appdata\roaming\python\python312\site-packages (2.3.2)
Requirement already satisfied: pyngrok in c:\users\as920\appdata\local\programs\python\python312\lib\site-packages (7.2.0)
Requirement already satisfied: filelock in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (3.13.4)
Requirement already satisfied: huggingface-hub<1.0,>=0.23.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (0.24.5)
Requirement already satisfied: numpy>=1.17 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (1.26.4)
Requirement already satisfied: packaging>=20.0 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (23.1)
Requirement already satisfied: pyyaml>=5.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (6.0.1)
Requirement already satisfied: regex!=2019.12.17 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (2023.6.3)
Requirement already satisfied: requests in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (2.31.0)
Requirement already satisfied: safetensors>=0.4.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (0.4.4)
Requirement already satisfied: tokenizers<0.20,>=0.19 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (0.19.1)
Requirement already satisfied: tqdm>=4.27 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (4.65.0)
Requirement already satisfied: python-dateutil>=2.8.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from pandas) (2023.3)
Requirement already satisfied: tzdata>=2022.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from pandas) (2023.3)
Requirement already satisfied: click in c:\users\as920\appdata\roaming\python\python312\site-packages (from nltk) (8.1.6)
Requirement already satisfied: joblib in c:\users\as920\appdata\roaming\python\python312\site-packages (from nltk) (1.3.1)
Requirement already satisfied: Werkzeug>=2.3.3 in c:\users\as920\appdata\roaming\python\python312\site-packages (from flask) (2.3.6)
Requirement already satisfied: Jinja2>=3.1.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from flask) (3.1.2)
Requirement already satisfied: itsdangerous>=2.1.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from flask) (2.1.2)
Requirement already satisfied: blinker>=1.6.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from flask) (1.6.2)
Requirement already satisfied: colorama in c:\users\as920\appdata\roaming\python\python312\site-packages (from click->nltk) (0.4.6)
Requirement already satisfied: fsspec>=2023.5.0 in c:\users\as920\appdata\roaming\python\python312\site-packages (from huggingface-hub<1.0,>=0.23.2->transformers) (2024.3.1)
Requirement already satisfied: typing-extensions>=3.7.4.3 in c:\users\as920\appdata\roaming\python\python312\site-packages (from huggingface-hub<1.0,>=0.23.2->transformers) (4.11.0)
Requirement already satisfied: MarkupSafe>=2.0 in c:\users\as920\appdata\roaming\python\python312\site-packages (from Jinja2>=3.1.2->flask) (2.1.3)
Requirement already satisfied: six>=1.5 in c:\users\as920\appdata\roaming\python\python312\site-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from requests->transformers) (3.2.0)
Requirement already satisfied: idna<4,>=2.5 in c:\users\as920\appdata\roaming\python\python312\site-packages (from requests->transformers) (3.4)
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from requests->transformers) (2.0.4)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\as920\appdata\roaming\python\python312\site-packages (from requests->transformers) (2023.7.22)
WARNING: There was an error checking the latest version of pip.
2024-10-13 07:21:34.536940: I tensorflow/core/util/port.cc:153] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-10-13 07:21:39.663501: I tensorflow/core/util/port.cc:153] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
Traceback (most recent call last):
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\server.py", line 2, in <module>
    import pandas as pd
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\pandas\__init__.py", line 22, in <module>
    from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\pandas\compat\__init__.py", line 25, in <module>
    from pandas.compat.numpy import (
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\pandas\compat\numpy\__init__.py", line 4, in <module>
    from pandas.util.version import Version
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\pandas\util\__init__.py", line 2, in <module>
    from pandas.util._decorators import (  # noqa:F401
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\pandas\util\_decorators.py", line 14, in <module>
    from pandas._libs.properties import cache_readonly
  File "C:\Users\as920\AppData\Roaming\Python\Python312\site-packages\pandas\_libs\__init__.py", line 13, in <module>
    from pandas._libs.interval import Interval
ModuleNotFoundError: No module named 'pandas._libs.interval'
Script finished. Check script_log.txt for any errors.
Requirement already satisfied: transformers in c:\users\as920\appdata\roaming\python\python312\site-packages (4.44.0)
Requirement already satisfied: pandas in c:\users\as920\appdata\local\programs\python\python312\lib\site-packages (2.2.1)
Requirement already satisfied: nltk in c:\users\as920\appdata\roaming\python\python312\site-packages (3.8.1)
Requirement already satisfied: textblob in c:\users\as920\appdata\local\programs\python\python312\lib\site-packages (0.18.0.post0)
Requirement already satisfied: flask in c:\users\as920\appdata\roaming\python\python312\site-packages (2.3.2)
Requirement already satisfied: pyngrok in c:\users\as920\appdata\local\programs\python\python312\lib\site-packages (7.2.0)
Requirement already satisfied: filelock in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (3.13.4)
Requirement already satisfied: huggingface-hub<1.0,>=0.23.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (0.24.5)
Requirement already satisfied: numpy>=1.17 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (1.26.4)
Requirement already satisfied: packaging>=20.0 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (23.1)
Requirement already satisfied: pyyaml>=5.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (6.0.1)
Requirement already satisfied: regex!=2019.12.17 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (2023.6.3)
Requirement already satisfied: requests in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (2.31.0)
Requirement already satisfied: safetensors>=0.4.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (0.4.4)
Requirement already satisfied: tokenizers<0.20,>=0.19 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (0.19.1)
Requirement already satisfied: tqdm>=4.27 in c:\users\as920\appdata\roaming\python\python312\site-packages (from transformers) (4.65.0)
Requirement already satisfied: python-dateutil>=2.8.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from pandas) (2023.3)
Requirement already satisfied: tzdata>=2022.7 in c:\users\as920\appdata\roaming\python\python312\site-packages (from pandas) (2023.3)
Requirement already satisfied: click in c:\users\as920\appdata\roaming\python\python312\site-packages (from nltk) (8.1.6)
Requirement already satisfied: joblib in c:\users\as920\appdata\roaming\python\python312\site-packages (from nltk) (1.3.1)
Requirement already satisfied: Werkzeug>=2.3.3 in c:\users\as920\appdata\roaming\python\python312\site-packages (from flask) (2.3.6)
Requirement already satisfied: Jinja2>=3.1.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from flask) (3.1.2)
Requirement already satisfied: itsdangerous>=2.1.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from flask) (2.1.2)
Requirement already satisfied: blinker>=1.6.2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from flask) (1.6.2)
Requirement already satisfied: colorama in c:\users\as920\appdata\roaming\python\python312\site-packages (from click->nltk) (0.4.6)
Requirement already satisfied: fsspec>=2023.5.0 in c:\users\as920\appdata\roaming\python\python312\site-packages (from huggingface-hub<1.0,>=0.23.2->transformers) (2024.3.1)
Requirement already satisfied: typing-extensions>=3.7.4.3 in c:\users\as920\appdata\roaming\python\python312\site-packages (from huggingface-hub<1.0,>=0.23.2->transformers) (4.11.0)
Requirement already satisfied: MarkupSafe>=2.0 in c:\users\as920\appdata\roaming\python\python312\site-packages (from Jinja2>=3.1.2->flask) (2.1.3)
Requirement already satisfied: six>=1.5 in c:\users\as920\appdata\roaming\python\python312\site-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\users\as920\appdata\roaming\python\python312\site-packages (from requests->transformers) (3.2.0)
Requirement already satisfied: idna<4,>=2.5 in c:\users\as920\appdata\roaming\python\python312\site-packages (from requests->transformers) (3.4)
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\users\as920\appdata\roaming\python\python312\site-packages (from requests->transformers) (2.0.4)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\as920\appdata\roaming\python\python312\site-packages (from requests->transformers) (2023.7.22)
WARNING: There was an error checking the latest version of pip.
2024-10-13 07:24:10.802273: I tensorflow/core/util/port.cc:153] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-10-13 07:24:12.432566: I tensorflow/core/util/port.cc:153] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
Traceback (most recent call last):
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\server.py", line 2, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
Script finished. Check script_log.txt for any errors.
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\myenv\Scripts\pip.exe\__main__.py", line 4, in <module>
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\myenv\Lib\site-packages\pip\_internal\cli\main.py", line 9, in <module>
    from pip._internal.cli.autocompletion import autocomplete
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\myenv\Lib\site-packages\pip\_internal\cli\autocompletion.py", line 10, in <module>
    from pip._internal.cli.main_parser import create_main_parser
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\myenv\Lib\site-packages\pip\_internal\cli\main_parser.py", line 8, in <module>
    from pip._internal.cli import cmdoptions
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\myenv\Lib\site-packages\pip\_internal\cli\cmdoptions.py", line 23, in <module>
    from pip._internal.cli.parser import ConfigOptionParser
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\myenv\Lib\site-packages\pip\_internal\cli\parser.py", line 12, in <module>
    from pip._internal.configuration import Configuration, ConfigurationError
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\myenv\Lib\site-packages\pip\_internal\configuration.py", line 26, in <module>
    from pip._internal.utils.logging import getLogger
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\myenv\Lib\site-packages\pip\_internal\utils\logging.py", line 27, in <module>
    from pip._internal.utils.misc import ensure_dir
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\myenv\Lib\site-packages\pip\_internal\utils\misc.py", line 39, in <module>
    from pip._internal.locations import get_major_minor_version
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\myenv\Lib\site-packages\pip\_internal\locations\__init__.py", line 14, in <module>
    from . import _distutils, _sysconfig
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\myenv\Lib\site-packages\pip\_internal\locations\_distutils.py", line 9, in <module>
    from distutils.cmd import Command as DistutilsCommand
ModuleNotFoundError: No module named 'distutils'
Traceback (most recent call last):
  File "C:\Users\as920\Desktop\mini_project\IR_PROJECT_CODE\server.py", line 1, in <module>
    from transformers import pipeline
ModuleNotFoundError: No module named 'transformers'
Script finished. Check script_log.txt for any errors.
